src/main/java/com/cloudant/client/org/lightcouch/CouchDbClient.java

```diff
@@ -89,11 +56,8 @@
  * .replicator()}
  * <p/>
  * <p/>
- * <p>At the end of a client usage; it's useful to call: {@link #shutdown()} to ensure proper
- * release of resources.
  *
  * @author Ahmed Yehia
- * @see CouchDbClientAndroid
  * @since 0.0.2
  */
 public class CouchDbClient extends CouchDbClientBase {
```

Comment:
As there is essentially nothing left here perhaps we should move the remaining `database()` method to `CouchDbClientBase` and then delete this class and rename `CouchDbClientBase` to `CouchDbClient`.
Wdyt?